lib/eco.js

```diff
@@ -808,23 +808,23 @@ class SimplyEco {
 		if (work !== null && timeout - (Date.now() - work) > 0) {
 			return {
 				error: 'ALREADY_WORKED',
-				timeout: msToTime(timeout - (Date.now() - work))
+				timeout: _msToTime(timeout - (Date.now() - work))
 			};
 		} else {
 			UserData.wallet += Number(AMT);
 			UserData.lastUsedWork = Date.now();
 			await UserData.save();
 			return {
 				wallet: UserData.wallet,
-				timeout: msToTime(work),
+				timeout: _msToTime(work),
 				job: UserData.job,
 				money: AMT
 			};
 		}
 	}
 }
 
-function msToTime(duration) {
+function _msToTime(duration) {
```

Comment:
I made `msToTime` a private function, because it is not important to any other extent.

Reply:
Alr got it